openwisp_notifications/tests/test_admin.py

```diff
@@ -250,6 +225,64 @@ def test_org_admin_view_same_org_user_notification_setting(self):
             response, '<option value="default" selected>Default Type</option>'
         )
 
+    def test_ignore_notification_widget_add_view(self):
+        url = reverse('admin:openwisp_users_organization_add')
+        response = self.client.get(url)
+        self.assertNotContains(response, 'owIsChangeForm')
+
+
+class TestAdminMedia(TestOrganizationMixin, TestMultitenantAdminMixin, TestCase):
+    """
+    Tests notifications admin media
+    """
+
+    def _login_admin(self):
+        u = User.objects.create_superuser('admin', 'admin', 'test@test.com')
+        self.client.force_login(u)
+        return u
+
+    def setUp(self):
+        self.admin = self._login_admin()
+        self.notification_options = dict(
+            sender=self.admin,
+            recipient=self.admin,
+            description='Test Notification',
+            verb='Test Notification',
+            email_subject='Test Email subject',
+            url='localhost:8000/admin',
+        )
+        self.site = AdminSite()
+        self.ns_inline = NotificationSettingInline(NotificationSetting, self.site)
+
+    @property
+    def _url(self):
+        return reverse('admin:index')
```

Comment:
avoid the duplication of the methods above by creating a base class called `BaseTestAdmin`.

Reply:
Done :+1: